050-Manage/060-migrations-multischema.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Multi Schema Migrations
+navTitle: Multi Schema Migrations
+keywords: ['pgroll']
+description: Learn about Multi Schema migration commands
+slug: multischema-migrations
+published: true
+status: beta
+---
+
+Multi Schema migrations address the pain point of having to keep your application code in sync with your database schema by allowing you two active database schemas at once. With Multi Schema migrations, running a migration means being able to preview the new version of your schema alongside the old one. Reading and writing data in both schemas continues to work normally. As a result of having two active schema versions, rollbacks become seamless operations which is particularly useful in the case of an unwanted schema change or backfill.
```

Comment:
It's probably worth highlighting and linking to pgroll int this intro blog, just so folks know what the underlying technology is

Reply:
Added. Just need to verify the link is correct